examples/platforms/nrf52811/README.md

```diff
@@ -0,0 +1,221 @@
+# OpenThread on nRF52811 Example
+
+This directory contains platform drivers for the Nordic Semiconductor nRF52811 SoC.
```

Comment:
This page will be displayed when one browses to the platform directory. That is why it is refered as 'this'.
We could say `examples/platforms/nrf52811/` but it would be less readable in our opinion.